documents/src/templates/import-element/ef-autosuggest.md

```diff
@@ -6,6 +6,6 @@ name: autosuggest
  * Import paths and theme-loader used for demonstration purposes
  */
```

Comment:
Interesting, can you go to the autosuggest section Header and footer?
I found a white background demo, don't know which purpose to set the white background on it

Reply:
background color removed